annotation/app/errors.py

```diff
@@ -51,6 +51,11 @@ def __init__(self, exc_info: Union[str, RequestException]):
         self.exc_info = exc_info
 
 
+class AgreementScoreServiceException(Exception):
+    def __init__(self, exc: Union[str, RequestException]):
```

Comment:
Is it okay behavior? I believe you should receive message only (`str`). Or if you want to save error type as well, you may want do this
```python
def __init__(self, message, errors):            
        # Call the base class constructor with the parameters it needs
        super().__init__(message)
            
        # Now for your custom code...
        self.errors = errors
```